### ISSUE TITLE
1st observation

### ISSUE THREAD

**Issue by perdolka** [piroor/aggregate-tabs-to-main-window#1]
This addon **can** open tabs in the main window **without** editing 
abut:config > browser.tabs.loadBookmarksInTabs > true
just open bookmarks by clicking mouse wheel (or middle mouse button).

**Comment by piroor** [comment]
Yes. Usage described at https://addons.mozilla.org/en-US/firefox/addon/aggregate-tabs-to-main-window/ is a special case. In this example, the window for Bookmarks sidebar is shrunken and bookmarks loaded in the content area will become invisible (due to too narrow window) and simple middle click on a bookmark will load the bookmark in the current invisible tab. `browser.tabs.loadBookmarksInTabs`=`true` avoids such a scenario.
